### PR TITLE
cramartef.md

### PR DESCRIPTION
**Pull request by cramartef**

### CODE REVIEW

**Comment by Yurockkk** [comment]
Hey @cramartef, It's a good start! try to add more markdown elements. check out [these](https://github.com/open-learning-exchange/open-learning-exchange.github.io/tree/master/pages/profiles) excellent examples

**Comment by cramartef** [comment]
Hi @Yurockkk , I updated my MD file [cramartef.md](https://github.com/open-learning-exchange/open-learning-exchange.github.io/pull/1166/commits/55ed14d08c4ff3aca07b27bf3f5aae75354f28ac) and add some markdown elements to it.  Please let me know your comment on it. Thank you

File: pages/profiles/cramartef.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Fetra Marc Humbert Rahajason  
+
+Madagascar/GMT+3.00, East Africa Time.  
+
+I am an OLE leader and Education consultant.
+---
+**Language skills**
+* Speak both french and English.
```
Comment (Yurockkk) [inline pages/profiles/cramartef.md]: @cramartef the layout seems a little bit wired on [rawgit](https://rawgit.com/cramartef/cramartef.github.io/Branch0-cramartef/#!pages/profiles/cramartef.md), try to add a line break between sub-heading and bucket list. Other than that, it looks good to me

File: pages/profiles/cramartef.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+* Translator/interpreter.
+
+**Programming and algorithms:**
+
+* learnt `MS-DOS, Pascal, C, C++, Matlab`
+* learning `HTML, CSS, latex, java`  
+---
+>I would like to help children especially disadvataged ones through education, give them equal chance to create their own future and improve their own lives.
+
+**E-mail:** [Fetra Marc Humbert Rahajason](humbert.fetra@outlook.com)
```
Comment (xyb994) [inline pages/profiles/cramartef.md]: The email link is not working. To mak the email link work as intended, you need to add `mailto:` before the email address:
```
[Fetra Marc Humbert Rahajason](mailto:humbert.fetra@outlook.com)
```
or use [autolink](https://daringfireball.net/projects/markdown/syntax#autolink):
```
<humbert.fetra@outlook.com>
```
Reference:
[Daring Fireball - Markdown: Syntax - Automatic Links](https://daringfireball.net/projects/markdown/syntax#autolink)
[GitHub Flavored Markdown Spec - Autolinks](https://github.github.com/gfm/#autolinks)

Comment (cramartef) [reply pages/profiles/cramartef.md]: Thanks @xyb994. I see.

**Pull request merged**